unittests/unittests_odcm.py

```diff
@@ -0,0 +1,290 @@
+############################################################################
+## Toolbox name: LargeNetworkAnalysisTools
+############################################################################
+'''Unit tests for the odcm.py module.'''
+################################################################################
+'''Copyright 2021 Esri
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.'''
+################################################################################
+
+import sys
+import os
+import datetime
+import unittest
+from copy import deepcopy
+
+import arcpy
+
+CWD = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.dirname(CWD))
+import odcm
+import portal_credentials  # Contains log-in for an ArcGIS Online account to use as a test portal

```

Comment:
@mmorang 
I prefer we include the portal_credentials.py file in the repo but without the credentials. The unit test should fail with invalid username and password error if the portal_credentials file does not contain valid credentials. 

I do not like the idea of having to do a lot more work in order to get the unit tests to run.